invoices/views.py

```diff
@@ -587,8 +635,11 @@ def invoice_page(request, invoice_id, **_):
 
 
     previous_invoices = []
+    recent_weekly_report = None
     if invoice.project_m:
         previous_invoices = Invoice.objects.filter(project_m=invoice.project_m)
+        weekly_reports = WeeklyReport.objects.filter(project_m=invoice.project_m, client=invoice.client)
```

Comment:
`Project` model always identifies client as well, so it should not be necessary to filter by `client` as well